src/main/java/jalau/at18/katas/bankocr/sarai/userstory1.java

```diff
@@ -0,0 +1,31 @@
+package jalau.at18.katas.bankocr.sarai;
+import java.util.Arrays;
+
+public class Userstory1 {
```

Comment:
`Userstory1` is not a good  class name, because is not part of the business logic of the problem, try to think in the class' goal (i.e. DigitReader)